### PR TITLE
Kike

### PR DESCRIPTION
**Pull request by kikemascote**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by kikemascote** [comment]
fixed

File: _data/Kikemascote/Kikemascote.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Enrique Mascote 
+institution: UTCH  
+profile_pic: Kike.jpg 
+quote: Let's get this work togheter
```
Comment (Mohammed-Shoaib) [inline _data/Kikemascote/Kikemascote.md:5]: Fix typo? Change `togheter` to `together`.

Comment (kikemascote) [reply _data/Kikemascote/Kikemascote.md]:
```suggestion
quote: Let's get this work together
```

done

**Comment by Mohammed-Shoaib** [comment]
Hey @kikemascote,

You modified a bunch of other files in your last commit! :no_good: You aren't supposed to do that, please note you can only modify 2 files:

1. The markdown file
2. The profile image

Both these files must be yours, please revert and undo the deletions. :memo:

Best,
Mohammed Shoaib

**Comment by kikemascote** [comment]
Sorry, I tought I've pushed extra files

**Comment by Mohammed-Shoaib** [comment]
@kikemascote Thanks for reverting back the deleted files, LGTM! :rocket:

**Comment by kikemascote** [comment]
Do I need to make anything else?

**Comment by Mohammed-Shoaib** [comment]
Hey @kikemascote,

Your pull request is perfect! :confetti_ball: I approved the changes. :+1:

However, I cannot merge it as I do not have write access. You'll have to wait for a reviewer with _write_ access to merge your pull request.

Till then sit back, relax, and enjoy :relaxed: 

Best,
Mohammed Shoaib

**Pull request merged**